ONSdigital/zebedee
# Release 1.4.1

### What
**Hotfix:** fixes issue where viewer encryption keys are being revoked from users who should have access to the collection when the collection details are updated.

### How to review
- Run the ONS Site stack locally.

- As an editor or publisher login to Florence create a collection & add a team of viewers to it.

- Make some change to the site content (_e.g._ edit a page).

- Login to Ermentrude as a user from your viewer team & you should see/have access to the collection. From within the collection select some content that has been added/edited. **You should be able to see the changes**.

- Login back into Florence edit & save the collection details _e.g._ change from manual to scheduled publish or vice versa.

- Login to Ermentrude as a user from your viewer team & you should still see/have access to the collection & be able to view content from the dropdown.
